cypress/e2e/simple-editing.cy.ts

```diff
@@ -1,14 +1,12 @@
+// Due to instabilities, this seems to occasionally fail.
+
 describe('EditorJS interaction', function() {
   // @ts-ignore
   beforeEach(() => cy.clearIndexedDB());
-  it('input content to EditorJS and retrieve the same content after reload', function() {
+  it('should input content to EditorJS and retrieve the same content after reload', function() {
     cy.visit('http://localhost:4200');
-    cy.get('.ce-header').click().type("Title"+"{enter}");
-    cy.get('.ce-paragraph').type("1{enter}");
-    cy.get('.ce-paragraph').eq(1).type("2{enter}");
-    cy.get('.ce-paragraph').eq(2).type("3{enter}");
-    cy.get('.ce-paragraph').eq(3).type("4{enter}");
-    cy.get('.ce-paragraph').eq(4).type("5");
+    cy.get('.ce-header').click().type("Title{enter}");
+    cy.get('.ce-paragraph').type("1{enter}2{enter}3{enter}4{enter}5");
```

Comment:
Okay, it makes sense and is more concise, but it requires an assertion.

Reply:
Which assertion do you suggest?